config/client.go

```diff
@@ -333,22 +333,6 @@ func GenerateLimitedClientConfig(c *model.Config, telemetryID string, license *m
 			props["SamlLoginButtonTextColor"] = *c.SamlSettings.LoginButtonTextColor
 		}
 
-		if *license.Features.CustomTermsOfService {
-			props["EnableCustomTermsOfService"] = strconv.FormatBool(*c.SupportSettings.CustomTermsOfServiceEnabled)
-			props["CustomTermsOfServiceReAcceptancePeriod"] = strconv.FormatInt(int64(*c.SupportSettings.CustomTermsOfServiceReAcceptancePeriod), 10)
-		}
-
-		if *license.Features.MFA {
-			props["EnforceMultifactorAuthentication"] = strconv.FormatBool(*c.ServiceSettings.EnforceMultifactorAuthentication)
-		}
-
-		if license.IsCloud() {
-			// MM-48727: enable SSO options for free cloud - not in self hosted
-			*license.Features.GoogleOAuth = true
-			*license.Features.Office365OAuth = true
```

Comment:
Will this change disable Google and Office 365 authentication in Cloud Free?

Reply:
If their license supports Google, O365, which is valid in case of Pro / Ent licenses, they can use this feature.

Reply:
I'm a little confused, doesn't the [ticket](https://mattermost.atlassian.net/browse/MM-49868) indicate that cloud free should have Google and Office 365 enabled?